packages/helpers/classes/mail.spec.js

```diff
@@ -8,10 +8,10 @@ const Mail = require('./mail');
 /**
  * Tests
  */
-describe('Mail', function() {
+describe('Mail', function () {
```

Comment:
Can you also undo these style corrections? Please respect the code style as indicated by the editor config and eslint config. If you enable ESLint in your editor it should automatically fix these issues.